source/testing_govuk_pay/index.html.md.erb

```diff
@@ -3,105 +3,65 @@ title: Testing GOV.UK Pay
 weight: 90
 ---
 
-# Testing GOV.UK Pay
+# Testing your integration
 
-When your GOV.UK Pay account is made live, you will still have a ‘test’
-account for general testing and experimenting with different settings and
-features. You can read more in the [__Switching to
-live__](/switching_to_live/#switching-to-live) section.
+### Testing your service
 
-When you test, you should make sure that you use [mock card
-numbers](#submit-a-test-transaction-using-mock-card-numbers) to simulate both successful
-and unsuccessful transactions. You must use test cards with your test account.
-Real card numbers will not work.
+You should:
 
-You should also:
+- run regular [smoke tests](https://www.gov.uk/service-manual/technology/deploying-software-regularly#using-smoke-tests-after-you-deploy) using your test account to check your service is always connected to GOV.UK Pay
+- build a stub of GOV.UK Pay's API, and use it to run frequent integration tests to check your service is making API requests correctly
 
- * only use your test account, not your live account
+You can use a library like [WireMock](https://github.com/WireMock-Net/WireMock.Net) to build a stub of GOV.UK Pay's API.
```

Comment:
also mention mountebank http://www.mbtest.org

Reply:
Thanks - added in https://github.com/alphagov/pay-tech-docs/pull/326/commits/041acc8d5a90525b0006afc2e2f2157fdd086b5e.